src/centralserver/admin-ui/src/views/AppLogin.vue

```diff
@@ -219,18 +216,15 @@ export default (
               });
             }
             this.$store.dispatch(
-              StoreTypes.actions.SHOW_ERROR_MESSAGE_CODE,
-              'login.generalError',
+              StoreTypes.actions.SHOW_ERROR_MESSAGE,
```

Comment:
Here we can be pretty sure that error is of type AxiosError , ie. we could use the .SHOW_ERROR, too.

Reply:
Fixed this